diff --git a/composer.json b/composer.json
index 31fc201..14de920 100644
--- a/composer.json
+++ b/composer.json
@@ -1,37 +1,37 @@
 {
-    "name": "sunnysideup/silverstripe-elemental-modeladmin",
-    "description": "An administration module for elemental content blocks",
-    "type": "silverstripe-vendormodule",
-    "keywords": [
-        "silverstripe",
-        "nsw",
-        "elemental",
-        "modeladmin"
-    ],
-    "license": "BSD-3-Clause",
-    "authors": [
-        {
-            "name": "James",
-            "homepage": "https://dpc.nsw.gov.au",
-            "role": "Developer"
-        }
-    ],
-    "require": {
-        "dnadesign/silverstripe-elemental": "^5 || ^6",
-        "symbiote/silverstripe-gridfieldextensions": "^4"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^9.5",
-        "friendsofphp/php-cs-fixer": "^3"
-    },
-    "autoload": {
-        "psr-4": {
-            "NSWDPC\\Elemental\\ModelAdmin\\": [
-                "src/"
-            ],
-            "NSWDPC\\Elemental\\ModelAdmin\\Tests\\": [
-                "tests/"
-            ]
-        }
+  "name": "sunnysideup/silverstripe-elemental-modeladmin",
+  "description": "An administration module for elemental content blocks",
+  "type": "silverstripe-vendormodule",
+  "keywords": [
+    "silverstripe",
+    "nsw",
+    "elemental",
+    "modeladmin"
+  ],
+  "license": "BSD-3-Clause",
+  "authors": [
+    {
+      "name": "James",
+      "homepage": "https://dpc.nsw.gov.au",
+      "role": "Developer"
     }
+  ],
+  "require": {
+    "dnadesign/silverstripe-elemental": "^6.0",
+    "symbiote/silverstripe-gridfieldextensions": "^5.2"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^9.5",
+    "friendsofphp/php-cs-fixer": "^3"
+  },
+  "autoload": {
+    "psr-4": {
+      "NSWDPC\\Elemental\\ModelAdmin\\": [
+        "src/"
+      ],
+      "NSWDPC\\Elemental\\ModelAdmin\\Tests\\": [
+        "tests/"
+      ]
+    }
+  }
 }
diff --git a/src/Controllers/ElementModelAdmin.php b/src/Controllers/ElementModelAdmin.php
index 5e82b04..9cc9393 100644
--- a/src/Controllers/ElementModelAdmin.php
+++ b/src/Controllers/ElementModelAdmin.php
@@ -2,6 +2,7 @@
 
 namespace NSWDPC\Elemental\ModelAdmin\Controllers;
 
+use Override;
 use DNADesign\Elemental\Models\BaseElement;
 use DNADesign\ElementalVirtual\Model\ElementVirtual;
 use SilverStripe\Admin\ModelAdmin;
@@ -61,6 +62,7 @@ class ElementModelAdmin extends ModelAdmin
      * Get the list of applicable elements, exclude ElementVirtual if available
      * @return DataList
      */
+    #[Override]
     public function getList()
     {
         $sort = (string) $this->config()->get('default_sort') ?: self::$default_sort;
@@ -75,6 +77,7 @@ class ElementModelAdmin extends ModelAdmin
         return $list;
     }
 
+    #[Override]
     public function getManagedModels()
     {
         $list = array_merge(
@@ -135,6 +138,7 @@ class ElementModelAdmin extends ModelAdmin
      * Return the GridField form listing elements
      * @return Form
      */
+    #[Override]
     public function getEditForm($id = null, $fields = null)
     {
         $form = parent::getEditForm($id, $fields);
diff --git a/src/Extensions/ElementSearchExtension.php b/src/Extensions/ElementSearchExtension.php
index 6e7cf13..9b380a6 100644
--- a/src/Extensions/ElementSearchExtension.php
+++ b/src/Extensions/ElementSearchExtension.php
@@ -76,7 +76,7 @@ class ElementSearchExtension extends Extension
             if ($sibling->ID === $owner->ID) {
                 $array[] = '<div style="padding-left: 2em;"><strong>&raquo; ' . $title . ' (this block) </strong></div>';
             } else {
-                $array[] = '<div style="padding-left: 2em;">- <a href="' . $sibling->CMSEditLink(true) . '">' . $title . '</a></div>';
+                $array[] = '<div style="padding-left: 2em;">- <a href="' . $sibling->getCMSEditLink(true) . '">' . $title . '</a></div>';
             }
         }
 
